application/backend/src/main/resources/apps/typerefinery/components/content/map/script.js

```diff
@@ -1,15 +1,51 @@
 $(document).ready(function (e) {
-  if (document.getElementById("map")) {
-    const key = "WTh8FTa6cT027rlVBBbG";
-    const map = new maplibregl.Map({
-      container: "map", // container's id or the HTML element in which MapLibre GL JS will render the map
-      style: `https://api.maptiler.com/maps/hybrid/style.json?key=${key}`, // style URL
-      center: [16.62662018, 49.2125578], // starting position [lng, lat]
-      zoom: 0, // starting zoom
-    });
-    map.addControl(new maplibregl.NavigationControl(), "top-right");
-    map.on("error", function (err) {
-      console.log("MAP ERROR", err)
+  const key = 'WTh8FTa6cT027rlVBBbG';
+  const dataId = 'dcb0900e-992a-4ff5-9819-861b4df293e9'
+  const dataId1= 'b70a0972-99a0-4539-98eb-35e04f0c052d'
+  const map = new maplibregl.Map({
+    container: 'map', // container's id or the HTML element in which MapLibre GL JS will render the map
```

Comment:
This script file will be included even if the map component does not exist in the UI, so please add the condition before executing the functionality,

Check for the map component and then proceed further.

Reply:
resolved.